qiskit_ionq/constants.py

```diff
@@ -63,4 +63,18 @@ class JobStatusMap(enum.Enum):
     FAILED = jobstatus.JobStatus.ERROR.name
 
 
+class AggregationType(enum.Enum):
```

Comment:
I'm meh on having this defined in the client.

Reply:
what do you mean?

Reply:
We might add more aggregation settings, it's weird to check that they're in an allowed list that's part of the client, because that would block new settings behind a client upgrade.

Reply:
I think sending an aggregation then returning a failed run with the error message specifying which aggregations are supported would be nice. we could then define those aggregations in openapi docs?